validator/.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what magic is this one? :D

Reply:
Husky added this on its own 😅 so I didn't touch it